src/Sulu/Bundle/PageBundle/Controller/PageController.php

```diff
@@ -643,6 +645,13 @@ private function getWebspaceNode(
         UserInterface $user
     ) {
         $webspace = $this->webspaceManager->findWebspaceByKey($webspaceKey);
+        if (null === $webspace->getLocalization($locale) || false === $this->securityChecker->hasPermission(
+            new SecurityCondition('sulu.webspaces.' . $webspace->getKey(), $locale, SecurityBehavior::class),
```

Comment:
Same here, use `PageAdmin::getPageSecurityContext($webspace->getKey())` instead of `'sulu.webspaces' . $webspace->getKey()`

Reply:
PR updated!